services/gafaelfawr/values-idfdev.yaml

```diff
@@ -1,3 +1,7 @@
+image:
+  pullPolicy: "Always"
+  tag: "tickets-DM-36458"
```

Comment:
Don't you want to change this prior to merge?

Reply:
...oh, or are you going to merge, and then do a followup PR with the actual tagged build so it exists when you're ready to sync?  That's assiduous.

Reply:
The pinning is just there so that I can test on data-dev from the branch, and then I'll rebase to drop the pins (which are in a separate commit) before I merge the PR branch.